src/main/java/com/bullhornsdk/data/model/entity/core/paybill/charge/PayExportBatch.java

```diff
@@ -89,22 +89,22 @@ public void setPayExportTypeLookup(SimplifiedOptionsLookup payExportTypeLookup)
     }
 
     @JsonProperty("payMasterTransactions")
-    public PayMasterTransaction getPayMasterTransactions() {
+    public OneToMany<PayMasterTransaction> getPayMasterTransactions() {
         return payMasterTransactions;
     }
 
     @JsonProperty("payMasterTransactions")
-    public void setPayMasterTransactions(PayMasterTransaction payMasterTransactions) {
+    public void setPayMasterTransactions(OneToMany<PayMasterTransaction> payMasterTransactions) {
         this.payMasterTransactions = payMasterTransactions;
     }
 
     @JsonProperty("payableCharges")
```

Comment:
Need @JsonIgnore here, see
https://github.com/bullhorn/sdk-rest/blob/86517577d303be32e8ed6411e71f84da00412e66/src/main/java/com/bullhornsdk/data/model/entity/core/standard/JobData.java#L381

Reply:
on the getter(s), not the setter(s)